### ISSUE TITLE
[BUG] Invalid encoding of API url (`&` char)

### ISSUE THREAD

**Issue by k4lizen** [hearchco/frontend#455]
![image](https://github.com/user-attachments/assets/ab5269ef-8003-456a-a18c-b75de9c34e6e)
Search query: `fear & hunger steam`

**Comment by aleksasiriski** [comment]
This might also be related to the fact that we're using Svetle 5 RC preview and not a stable build. But it also might not be related to that :)

**Comment by aleksasiriski** [comment]
LGTM now, can you confirm @k4lizen ?

**Comment by k4lizen** [comment]
yup, after clearing the site's storage in my browser it's working.
